docs/Data_correction_documentary_cleared.ipynb

```diff
@@ -0,0 +1,290 @@
+    "\n",
+    "To facilitate an in-situ data correction during the experiments, we have implemented two different approaches: (i) a rough data-correction based on a <sup>3</sup>He-cell opacity calculated by its cell parameters and filling pressure (\"workflow 1\") which can be used in-situ during the experiment, and (ii) a precise calculation based on the calculation of opacity by a direct-beam measurement on the depolarized cell at its end of lifetime (i.e., end of cell-usage for this experiment) (\"workflow 2\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "405b0f3e-67ab-43d8-80de-566a1b93785a",
+   "metadata": {},
+   "source": [
+    "### Data correction"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "159bc48e",
+   "metadata": {},
+   "source": [
+    "#### Calculation of <sup>3</sup>He opacity \n",
+    "The <sup>3</sup>He opacity is a cell-specific parameter and can vary after each filling process. Hence, for precise data correction, it must be measured for each newly filled cell. It linearly depends on the neutron wavelength as follows:\n",
+    "\n",
+    "$$\n",
+    "O^{cell}(\\lambda)=O^{0, cell}\\cdot\\lambda\n",
+    "$$\n",
+    "\n",
+    "where the wavelength-independent parameter $O^{0, cell}$ depends on cell properties and filling pressure. In the following, two different approaches are presented to measure $O^{0, cell}$:"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "add255a3-3222-444a-b2e6-b8cef3a8b2db",
+   "metadata": {
+    "jp-MarkdownHeadingCollapsed": true
+   },
+   "source": [
+    "$\\mathbf{Workflow 1}$:<br>\n",
+    "For an in-situ quick data correction, a measure of the cell pressure $p$ at 20C during filling the cell can be used to calculate $O^{0, cell}$ by following relation (Ref 1):\n",
+    "\n",
+    "$$\n",
+    "O^{0, cell}= 0.0733 \\cdot p^{cell} \\cdot l^{cell}\n",
+    "$$\n",
+    "\n",
+    "with a factor 0.0733 arising from the number density of <sup>3</sup>He gas, the path length $l$ of neutron through the cell, and the absorption cross section of 1$\\mathring A$ neutrons by <sup>3</sup>He. However, a measurement of the opacity by neutron transmission in a depolarized cell condition yields more precise values and can be performed at the end of the cell-lifetime during the user's neutron experiments, i.e., leading to \"workflow 2\". \n",
```

Comment:
Is this 0.0733 missing a unit? I can't get the opacity math to work otherwise.

Reply:
Yes, it has. To get this pre-factor according to the pressure [bar], cell length [cm] and wavelength [Angstrom] it is 0.0733[1/(kg/s2)] pro Angstrom. I mentioned it in the code of the branch calculating O, but yes of course I should better insert that here in the documentary: O is calculated from O=n*sigma*lambda*l ("n is the number density of 3He gas in the cell. σ0 is the absorption cross section of 1 Å neutron by 3He, σ0 = 2966 × 10−24 cm2. λ is the neutron wavelength. l is the path length through the cell. " - from Hals paper of the ECNS proceedings). Then exchanging with p = nk_BT. I thought its ok as its reported in other papers but its a quick copy-paste!

Reply:
> 0.0733[1/(kg/s2)] pro Angstrom

Where is the Angstrom in that fraction, exactly?

Reply:
If we can define things in terms of other constants that would be even better, avoiding magic numbers.

Reply:
Please see my updates in https://github.com/scipp/esspolarization/pull/27.

Reply:
Yes I am having a look on the changes now - trying to keep track after coming back from USA! If I forget anything there please dont hesitate to remind me again.